app/models/queue_item.rb

```diff
@@ -0,0 +1,24 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :video
+
+  validates_presence_of :user
+  validates_uniqueness_of :position, scope: :user_id
```

Comment:
Got it.  Yeah, I'm running into issues with it right now, working on the queue form.  I actually had to turn it off to get things to work.